tests/test_errata.py

```diff
@@ -422,6 +422,82 @@ def test_errata_get_cve_affected_rpm_nvrs(self, errata_http_get, errata_rest_get
         ret = self.errata.get_cve_affected_rpm_nvrs(28484)
         self.assertEqual(ret, ['libntirpc-1.4.3-4.el6rhs'])
 
+    @patch.object(Errata, "_errata_rest_get")
+    @patch.object(Errata, "_errata_http_get")
+    def test_errata_get_cve_affected_build_nvrs(self, errata_http_get, errata_rest_get):
+        mocked_errata = MockedErrataAPI(errata_rest_get, errata_http_get)
+        mocked_errata.builds_by_cve = {
```

Comment:
I would rather make short excessive lists of rpms, but use more items in builds and the main dictionary. They can also have different values of 'is_module', so you can test both calls with the second argument `True` and `False`.

Reply:
This test data is tweaking from the real data, actually I am not sure about all the build data format, it will cost some time for me to find a build with both module and other content types(I don't even know if that really exists or not).  So I only focus on module build data and its testing. I think if we really meet this use case, we can add more test cases in future.

Reply:
I didn't mean that you should run all the data in one run. You can make a couple of runs with different samples of data if you want. In the unit tests, you should cover as many branches as possible. It doesn't really matter if your test data "really exists" or not.

Reply:
Yes. I agree that test coverage is important, for this patch, it is simple to add and I am fine, but I am still wondering if that makes sense to add a non-existing case with lots of effort to mock it. Just want to align everyone on the same page @qixiang @hluk

Reply:
Tests updated

Reply:
Thank you @chuazhan